components/myforms/Tabs/Tabs.test.js

```diff
@@ -44,7 +44,7 @@ describe("Tabs component", () => {
     render(
       <>
         <h2 id={tabsData.id}>{tabsData.title}</h2>
```

Comment:
Renamed Tabs to TabsList to avoid confusion. Looks like I deleted the file and recreated it vs. renaming it. Renaming it would have made more sense but this works to.